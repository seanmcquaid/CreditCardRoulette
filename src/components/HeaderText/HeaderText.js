import {Text} from "react-native";
import styled from "styled-components/native";

export default styled(Text)`
    font-family : "FiraSans"
`;