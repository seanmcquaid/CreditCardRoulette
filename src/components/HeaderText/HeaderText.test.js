import React from "react";
import {render} from "react-native-testing-library";
import HeaderText from "./HeaderText";

describe("<HeaderText/>", () => {
    it("Matches snapshot correctly", () => {
        const headerText = render(<HeaderText>Header Text</HeaderText>);
        expect(headerText).toMatchSnapshot();
    });
});