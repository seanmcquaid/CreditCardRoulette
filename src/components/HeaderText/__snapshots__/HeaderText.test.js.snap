// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderText/> Matches snapshot correctly 1`] = `
<Text
  style={
    Array [
      Object {
        "fontFamily": "Fira-Sans",
        "fontSize": 26,
        "marginBottom": 10,
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 10,
      },
    ]
  }
>
  Header Text
</Text>
`;
